src/Shorttag/DownloadUrl.php

```diff
@@ -1,19 +1,19 @@
 <?php
 
-namespace GFExcel\Shorttag;
+namespace GFExcel\Shortcode;
 
 use GFExcel\Addon\GravityExportAddon;
 use GFExcel\GFExcel;
 
 /**
- * A short tag handler for [gravityexport_download_url].
+ * A shortcode handler for [gravityexport_download_url].
  * Example usage: [gravityexport_download_url id=1 type=csv]
  * Id is required, type is optional.
  * @since 1.6.1
  */
 class DownloadUrl {
 	/** @var string */
-	public const SHORTTAG = 'gravityexport_download_url';
+	public const SHORTCODE = 'gravityexport_download_url';
```

Comment:
This is also a breaking change. We need to at least keep the old `const` around (deprecated)